test/vol.c

```diff
@@ -558,6 +561,75 @@ reg_opt_datatype_get(void H5_ATTR_UNUSED *obj, H5VL_datatype_get_args_t *args, h
     return ret_value;
 } /* end reg_opt_datatype_get() */
 
+/*-------------------------------------------------------------------------
+ * Function:    fake_vol_info_to_str
+ *
+ * Purpose:     Convert the fake VOL info to a string
+ *
+ * Return:      SUCCEED/FAIL
+ *
+ *-------------------------------------------------------------------------
+ */
+static herr_t
+fake_vol_info_to_str(const void *info, char **str)
+{
+    herr_t    ret_value = SUCCEED; /* Return value */
+    const int val       = *(const int *)info;
+
+    /* Verify the info is correct before continuing */
+    if (val != INT_MAX) {
```

Comment:
This new check is just going to fail since INT_MAX isn't what's being passed in here. The changes failed right away for the Ubuntu action, though I don't know why that isn't showing up for this PR.

Reply:
Fixed.  I thought I changed the info value to INT_MAX.